app/wizards/placements/edit_placement_wizard/provider_step.rb

```diff
@@ -0,0 +1,15 @@
+class Placements::EditPlacementWizard::ProviderStep < Placements::BaseStep
+  attribute :provider_id
+
+  validates :provider_id, inclusion: { in: ->(step) { step.providers_for_selection.ids } }, if: -> { provider_id != "on" }
```

Comment:
What's the significance of `"on"` here? What does it represent/mean?

Reply:
"on" is returned by a radio button when there is no value, e.g. `nil` this is expected behaviour. I could set an explicit value if you prefer?

Reply:
Add "not_known" constant and use that